nep-25.mediawiki

```diff
@@ -0,0 +1,100 @@
+<pre>
+  NEP: 25
```

Comment:
We also need to add the corresponding entries to the README.

Reply:
Readme can only be updated after this is merged.

Reply:
Or at same time 🤣

Reply:
> Or at same time 🤣

@shargon  Same problem here,,,,, you have approves, if we update the pr, all approves dismissed..... If you approve my another NEP for name rules, then we can directly update readme.